_posts/2019-05-20-server-rendering-responsively.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: epic
+title: "Server-Rendering Responsively"
+date: "2019-05-20"
+author: [steve-hicks]
+categories: [react, html, web]
+---
+
+We use server-side rendering (SSR) to deliver every page you hit on [artsy.net](https://artsy.net). We decided on
+using SSR for many reasons, amongst them performance.
+[We wrote about this all the way back in 2013](https://artsy.github.io/blog/2013/11/30/rendering-on-the-server-and-client-in-node-dot-js/)!
+
+We've also built our site using responsive design, so you get a browsing experience optimized for your device.
+
+Combining SSR and responsive design is a non-trivial problem. There are many concerns to manage, and they are
+sometimes in conflict with each other. We server render for performance reasons, but we also want to be sure our
+app is performant when your browser takes over, all while optimizing for accessibility and SEO.
+
+This article describes the tools we use on [artsy.net](https://artsy.net) to combine SSR and responsive design.
+
+<!-- more -->
+
+## Tool 1: [`styled-system`](https://styled-system.com)
+
+We handle the majority of responsive styling differences with
+[`styled-system`](https://styled-system.com/responsive-styles). This has been a really great addition to our
+toolbox. Here's a component that would render a `div` (`Box`) with a width of 50% at our mobile breakpoint, 75% at
+our tablet breakpoint, and 100% for anything larger:
```

Comment:
We don’t actually talk about ‘mobile’ and ‘tablet’ breakpoints afaik and [below](https://github.com/artsy/artsy.github.io/pull/566/files#diff-a879e573d311218fd7eec7da14d776f5R81) you mention `xs`. Perhaps there’s a way to reword this without the device references?